client/verta/tests/test_entities.py

```diff
@@ -323,8 +323,9 @@ def test_get(self, client):
         assert run.id == client.get_experiment_run(run.name).id
         assert run.id == client.get_experiment_run(id=run.id).id
 
-        # clear client state to test setting parent
-        client.proj = client.expt = None
+        # test parents are restored
+        client.set_project()
```

Comment:
Shouldn't these be restored automatically?

Reply:
Yes. This test is setting a new Project and Experiment, and then testing that `get_experiment_run()` restores the _old_ ones.

Reply:
Fair enough.